### PR TITLE
several minor updates (see description)

### PR DESCRIPTION
**Pull request by LeonHoog**
- Minor updates to Dutch language strings.
- Removed an await syntax in main.dart since the app won't open if there are any issues with the indexing process.
- Fix to issue caused by PR #112: wrong image called in collection.dart
- removed unused imports
- I've moved the miscellaneous & themeMode settings over to a Hive database. This still needs some work, for example if a settings depends on custom classes it'll need a TypeAdapter.

### CODE REVIEW

**Comment by alexmercerind** [comment]
Thanks a lot for the PR. I appreciate your contribution.

There's a personal local branch that I'm working on as part of completely rewriting the indexing engine from scratch. I will give review/merge soon.

Appreciate your work.

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -36,10 +36,10 @@
                  screen fades out. A splash screen is useful to avoid any visual
                  gap between the end of Android's launch screen and the painting of
                  Flutter's first frame. -->
-            <meta-data
+            <!--<meta-data
```
Comment (alexmercerind) [inline android/app/src/main/AndroidManifest.xml:39]: Is this causing some problem? Won't remove Android splash screen?

File: lib/main.dart [hunk 2]
```diff
@@ -41,16 +42,18 @@ const String AUTHOR = 'Hitesh Kumar Saini <saini123hitesh@gmail.com>';
 const String LICENSE = 'GPL-3.0';
 
 Future<void> main(List<String> args) async {
+  await Hive.initFlutter();
+  var configurationBox = await Hive.openBox('configuration');
```
Comment (alexmercerind) [inline lib/main.dart:46]: Ideally, I would like to move all `Hive` related method calls entirely into `Configuration` class.
Current `Configuration` class is actually really helpful in giving good intellisense, and one doesn't mess up with wrong key/box strings. A single `update` method does it all.

I can do it in this PR itself or you are free to do it. I'm already happy with what you've done. 🙂

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (LeonHoog) [reply android/app/src/main/AndroidManifest.xml:39]: This doesn't cause any problem, but Flutter says this function is deprecated.
https://flutter.dev/docs/development/ui/advanced/splash-screen#migrating-from-manifest--activity-defined-custom-splash-screens

Comment (alexmercerind) [reply android/app/src/main/AndroidManifest.xml:39]: Thanks. I will take look, I didn't attempt to run on Android after Flutter update (2.5?)

**Comment by alexmercerind** [comment]
Are you yet to push anything @LeonHoog ?

**Comment by LeonHoog** [comment]
I'm making a few changes to this PR. This way everything related to Hive will be done inside configuration.dart

**Pull request closed without merging**